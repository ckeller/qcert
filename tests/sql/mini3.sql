select
	distinct( l_returnflag )
from
	lineitem
where
	l_shipdate <= date '1993-12-01';