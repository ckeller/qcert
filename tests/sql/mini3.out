[{"l_returnflag" : "A"},
 {"l_returnflag" : "R"}]